CMakeLists.txt

```diff
@@ -303,6 +303,7 @@ if(MSVC)
     # Enable UTF-8 support
     add_compile_options( $<$<C_COMPILER_ID:MSVC>:/utf-8> )
     add_compile_options( $<$<CXX_COMPILER_ID:MSVC>:/utf-8> )
+    add_compile_options($<$<OR:$<COMPILE_LANGUAGE:C>,$<COMPILE_LANGUAGE:CXX>>:-Wno-cast-function-type-mismatch>)
```

Comment:
I'd prefer to use the proper casts (whatever that is) rather than disable the warning.

Give that warnings were previously disabled around this code for MSVC how come we started getting them with the latest compiler?

Reply:
> Give that warnings were previously disabled around this code for MSVC how come we started getting them with the latest compiler?

Oh! Were the failures in the MingW build using GCC?